tests/specs/v2/rumXhrStatusTracking.js

```diff
@@ -4,8 +4,16 @@ var _ = require('underscore');
 
 describe("RUM status code tracking", function() {
 
-  afterEach(function() {
-    browser.reloadSession();
+  beforeEach(function() {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
```

Comment:
what is the test execution time impact of this change?

Reply:
Not much this only applies for these few tests. and most of them complete will within the timeout period.

It takes about 5 min to complete all of the tests

Reply:
To took about 5 min the last time these tests were run successfully

Reply:
Ok i have made this only pause as long as it needs to (up to 40 seconds)

Reply:
sounds good, seeing as these are only being run by GH actions this won't have an impact on any TC builds. Happy for the added test time due to the async behaviour 😄